build/postgres-gis-ha/Dockerfile

```diff
@@ -95,6 +95,9 @@ RUN chmod +x /opt/crunchy/bin/yq
 # volume permissions are applied when building the image
 VOLUME ["/pgdata", "/pgwal", "/pgconf", "/backrestrepo", "/sshd"]
 
+# Defines a unique directory name that will be utilized by the nss_wrapper in the UID script
+ENV NSS_WRAPPER_SUBDIR="postgis-ha"
+
```

Comment:
Are we unable to leverage this from the `postgres-gis` container?

Reply:
In general that should be fine, since uniqueness is really to avoid conflicts when two containers are running in the same Pod, which obviously wouldn't happen with `postgres-gis` & `postgres-gis-ha`.

However, I felt that establishing unique directories per-container keeps us safe regardless of how these containers are run, so I simply decided to add a unique value for this container as well.

Reply:
OK.

Reply:
Actually I take this back -- Looking at https://github.com/CrunchyData/postgres-operator/pull/2356/files#diff-9438712811604f4e58616aad9619188a102937147f58ec5745bb9903f55d0284R155 this seems like it will break things.

Reply:
Thansk, great catch - I will adjust to use the same name across _all_ PG containers.

Reply:
The `crunchy-postgres-ha`, `crunchy-postgres-gis` and `crunchy-postgres-gis-ha` containers now all simply inherit the `NSS_WRAPPER_SUBDIR` variable defined in the `postgres` Dockerfile.